packages/flutterfire_cli/lib/src/commands/config.dart

```diff
@@ -126,6 +126,27 @@ class ConfigCommand extends FlutterFireCommand {
       help:
           'Whether to generate the firebase_app_id.json files used by native iOS and Android builds.',
     );
+
+    argParser.addOption(
+      'ios-out',
+      valueHelp: 'pathForIosConfig',
+      help:
+          'Where would you like your `Google-Service-Info.plist` file to be written for iOS platform. Useful for different flavors',
```

Comment:
```suggestion
          'Where to write the `Google-Service-Info.plist` file for the iOS platform. Useful for different flavors',
```

I would use a shorter sentence, less polite 😅